tests/test_irbempy.py

```diff
@@ -237,6 +237,14 @@ def test_coord_trans(self):
             [ 1.91462214,  0.06992421,  0.57387514]])
         numpy.testing.assert_almost_equal(expected, ib.coord_trans(self.loci, 'GSM', 'car'))
 
+    def test_GSM_SM_init(self):
+        '''test for initialization error in gsm to sm conversion'''
+        cc_got = ib.oplib.coord_trans1(2, 4, 2002, 33, 43200, np.asarray([1., 2., 4.]))
+        expected = np.array([1.9286, 2., 3.6442])
+        # NaN will result if init not done in IRBEM, assert_almost_equal will
+        # compare NaNs without complaint
+        numpy.testing.assert_almost_equal(expected, cc_got, decimal=3)
+
```

Comment:
Confirmed this fails on master (and presumbly succeeds on PR or the CI would fail)